config/queue.php

```diff
@@ -39,8 +39,11 @@
             'table' => 'jobs',
             'queue' => 'default',
             'retry_after' => 90,
```

Comment:
Is this definietly in seconds? I'm just thinking about our over retry disaster.

Reply:
yep, it's a default in seconds.